contrib/i2c-tools/template.py

```diff
@@ -0,0 +1,31 @@
+pkgname = "i2c-tools"
+pkgver = "4.3"
+pkgrel = 0
+build_style = "makefile"
+make_cmd = "gmake"
+make_build_args = ["EXTRA=py-smbus"]
+make_install_args = ["sbindir=/usr/bin"] + make_build_args
+hostmakedepends = ["gmake", "python", "python-setuptools"]
+makedepends = ["linux-headers", "python-devel"]
+depends = ["perl"]
+pkgdesc = "Heterogeneous set of I2C tools and I2C library for Linux"
+maintainer = "Subhaditya Nath <sn03.general@gmail.com>"
+license = "GPL-2.0-or-later"
```

Comment:
this should be specified as an spdx license expression covering all the licenses of the project

Reply:
so... should it be `AND` or `OR` ?

Reply:
AND is for mixed licenses, OR is for dual/triple/etc licensing

Reply:
so.... `AND`, right?

Reply:
yes